src/templates/page.js

```diff
@@ -156,45 +167,50 @@ export default ({ data: { mdx, allMdx } }) => {
     items: versions
   };
 
+  console.log("customSlug", customSlug);
```

Comment:
;)

Reply:
just testing on feature branch (changing the URL structure affects functionality)

Reply:
ok I will use gatsby serve